package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "codeclimate-fixme",
+  "description": "Static analysis tool that finds FIXME, TODO, BUG, etc. comments in your code.",
+  "version": "0.0.1",
+  "main": "./lib/fix-me.js",
+  "dependencies": {
+    "glob": "6.0.1"
+  },
+  "devDependencies": {
+    "chai": "3.4.1",
+    "mocha": "2.3.3"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "files": [
+    "LICENSE",
+    "README.md",
+    "bin",
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/codeclimate/codeclimate-fixme"
```

Comment:
This should be the git url, so `https://github.com/codeclimate/codeclimate-fixme.git`